hw_11/HW_1_Sotnikov.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python3
+
+# 1 Chat-message-user
+from datetime import datetime
+
+## Creation class chat
+class Chat(list):
+    def __init__(self, container):
+        self.chat_history = []
+
+
+    def show_last_message(self, number=0):
```

Comment:
last это last, аргумент не нужен

```suggestion
    def show_last_message(self):
```